pkg/api/vminfo.go

```diff
@@ -328,14 +328,27 @@ type ScalingConfig struct {
 	// this field is left out the settings will fall back on the global default.
 	LoadAverageFractionTarget *float64 `json:"loadAverageFractionTarget,omitempty"`
 
-	// MemoryUsageFractionTarget sets the desired fraction of current memory that
-	// we would like to be using. For example, with a value of 0.7, on a 4GB VM
-	// we'd like to be using 2.8GB of memory.
+	// MemoryUsageFractionTarget sets the maximum fraction of current memory that we'd like to allow
+	// to be allocated (exactly MemTotal - MemAvailable, or roughly the memory that postgres has
+	// allocated [notably excluding LFC]).
+	// This memory may also be viewed as "unreclaimable" (contrary to e.g. page cache).
+	//
+	// For example, with a value of 0.75 on a 4GiB VM, we will try to upscale if the unreclaimable
+	// memory usage exceeds 3GiB.
 	//
 	// When specifying the autoscaler-agent config, this field is required. For an individual VM, if
 	// this field is left out the settings will fall back on the global default.
 	MemoryUsageFractionTarget *float64 `json:"memoryUsageFractionTarget,omitempty"`
 
+	// MemoryTotalFractionTarget sets the maximum fraction of current memory that we'd like to allow
+	// to be consumed by allocated memory (MemTotal minus MemAvailable) plus short-term cached
+	// memory by the LFC.
+	//
+	// This is kind of parallel to MemoryUsageFractionTarget, where the difference is that the
+	// "usage" target mostly tries to makes sure there's plenty of memory for postgres, whereas the
+	// "total" target also takes LFC cached memory into account.
```

Comment:
```suggestion
	// MemoryTotalFractionTarget sets the maximum fraction of total memory that postgres allocations PLUS LFC memory (MemoryUsage + MemoryCached) must fit into. 
	//
	// Compared with MemoryUsageFractionTarget, this value can be set higher (e.g. 0.9 vs 0.75), because we can tolerate higher fraction of consumption for both in-VM memory consumers.
```